Automattic/vip-design-system
# Upgrade storybook to 6.5.10

## Description

Upgraded StoryBook to the newest version.

![image](https://user-images.githubusercontent.com/3402/182948817-a54527b5-fc46-48c2-8bd1-b3f1a624d91a.png)

## Checklist

N/A

## Steps to Test

1. Pull down PR.
2. `npm run dev`.
3. Open Storybook.
4. Everything should be working correctly
